src/handlers.cpp

```diff
@@ -329,19 +329,20 @@ void ImpiTask::send_mar()
 
 void ImpiTask::on_mar_response(Diameter::Message& rsp)
 {
-  Cx::MultimediaAuthAnswer maa(rsp);
+  Cx::MultimediaAuthAnswer *maa = new Cx::MultimediaAuthAnswer(rsp);
```

Comment:
Why don't we put the Cx::MultimediaAuthAnswer in the ImpiTask itself?  That would avoid having to remember to delete the Cx::MultimediaAuthAnswer when we're finished.